app/scripts/controllers/gene.js

```diff
@@ -3595,12 +3580,8 @@ angular.module('oncokbApp')
                 if (_.isUndefined(data.tumorRef.cancerTypes_review)) {
                     data.tumorRef.cancerTypes_review = {};
                 }
-                if (_.isUndefined(data.tumorRef.cancerTypes_review.updatedBy)) {
```

Comment:
Why did we remove these checks?

Reply:
In this case, it won't record the updatedBy and updateTime after first initialization.